### PR TITLE
Add email validation in person new and edit

### PR DESCRIPTION
**Pull request by lkleisa**
Current state is described here: https://github.com/puzzle/skills/issues/497

### CODE REVIEW

File: frontend/app/components/person-new.js [hunk 1]
```diff
@@ -99,7 +100,7 @@ export default ApplicationComponent.extend({
               `person.${attribute}`
             );
             this.get("notify").alert(`${translated_attribute} ${message}`, {
-              closeAfter: 8000
+              closeAfter: config.environment === "test" ? null : 8000
```
Comment (Robin481) [inline frontend/app/components/person-new.js:103]: We should probably do this globally as the notifications are seemingly broken in all tests?

File: frontend/tests/acceptance/create-person-test.js [hunk 2]
```diff
@@ -77,4 +79,66 @@ module("Acceptance | create person", function(hooks) {
     assert.equal(currentURL(), "/people/new");
     // TODO expect errors!
   });
+
+  test("should display two error when email is empty", async function(assert) {
```
Comment (Robin481) [inline frontend/tests/acceptance/create-person-test.js:83]: error**s**

**Review by Robin481 — CHANGES_REQUESTED** [review]

**Comment by Robin481** [comment]
Looking good :partying_face: implement small feedback and we should be good

File: frontend/app/components/person-new.js (hunk 1 above)
Comment (lkleisa) [reply frontend/app/components/person-new.js:103]: https://github.com/puzzle/skills/issues/512

**Comment by lkleisa** [comment]
Follow up issue: https://github.com/puzzle/skills/issues/512

**Review by Robin481 — APPROVED** [review]

**Pull request merged**